libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -305,7 +304,12 @@ singleton QuadPlane depends APM_BUILD_TYPE(APM_BUILD_ArduPlane)
 singleton QuadPlane method in_vtol_mode boolean
 
 include AP_Motors/AP_MotorsMatrix.h
-singleton AP_MotorsMatrix depends APM_BUILD_TYPE(APM_BUILD_ArduPlane)||APM_BUILD_TYPE(APM_BUILD_ArduCopter) 
+singleton AP_MotorsMatrix depends APM_BUILD_TYPE(APM_BUILD_ArduPlane)||APM_BUILD_TYPE(APM_BUILD_ArduCopter)
 singleton AP_MotorsMatrix alias MotorsMatrix
 singleton AP_MotorsMatrix method init boolean uint8_t 0 (AP_MOTORS_MAX_NUM_MOTORS-1)
 singleton AP_MotorsMatrix method add_motor_raw void int8_t 0 (AP_MOTORS_MAX_NUM_MOTORS-1) float -FLT_MAX FLT_MAX float -FLT_MAX FLT_MAX float -FLT_MAX FLT_MAX uint8_t 0 (AP_MOTORS_MAX_NUM_MOTORS-1)
+
+include AP_Frsky_Telem/AP_Frsky_SPort.h
+singleton AP_Frsky_SPort alias frsky_sport
```

Comment:
we could put this behind the new depends stuff,
```suggestion
singleton AP_Frsky_SPort alias frsky_sport depends HAL_WITH_FRSKY_TELEM_BIDIRECTIONAL
```

Reply:
if it needs it of course....

Reply:
incorrect dependency